monkey/monkey_island/cc/main.py

```diff
@@ -55,5 +55,27 @@ def main():
         IOLoop.instance().start()
 
 
+def wait_for_mongo_db_server(mongo_url):
+    while not is_db_server_up(mongo_url):
+        logger.info('Waiting for MongoDB server on {0}'.format(mongo_url))
+        time.sleep(1)
+
+
+def assert_mongo_db_version(mongo_url):
```

Comment:
Can we provide a possible screen for the user? Make a simple HTTPS page server that returns an error message?

Reply:
We can do it, but I think it might take way too long to make it work, and it's not worth it.

Reply:
Also, not a blocker for this PR, but rather can be added to the backlog ⌚️

Reply:
Ok